server/src/main/java/org/elasticsearch/ElasticsearchException.java

```diff
@@ -969,7 +969,12 @@ private enum ElasticsearchExceptionHandle {
             org.elasticsearch.cluster.coordination.NodeHealthCheckFailureException.class,
             org.elasticsearch.cluster.coordination.NodeHealthCheckFailureException::new,
             175,
-            Version.V_5_2_0);
+            Version.V_5_2_0),
+        UNAUTHORIZED_EXCEPTION(
+            io.crate.exceptions.UnauthorizedException.class,
+            io.crate.exceptions.UnauthorizedException::new,
+            177,
+            Version.V_5_4_8);
```

Comment:
Want to double check, that this is the correct way, to use the snapshot version on every branch, or if we should use the lower one (I think  is 5.3.9).

Reply:
Yes this will be the new release which will support streaming of this exception. If I would use `5.3.9` and then I run a mixed one with e.g. `5.4.2`, it would expect it to work as `5.4.2` >  `5.3.9`.

Reply:
Correct, thx! and if we do a `5.3.9` -> `5.4.8` it will also work, as the exception is valid for both versions.